docs/guides/guides/command-line.mdx

```diff
@@ -121,7 +121,7 @@ cypress run [options]
 | `--env`, `-e`                  | [Specify environment variables](#cypress-run-env-lt-env-gt)                                                                                                                                   |
 | `--group`                      | [Group recorded tests together under a single run](#cypress-run-group-lt-name-gt)                                                                                                             |
 | `--headed`                     | [Displays the browser instead of running headlessly](#cypress-run-headed)                                                                                                                     |
-| `--headless`                   | [Hide the browser instead of running headed (default during `cypress run`)](#cypress-run-headless)                                                                                            |
+| `--headless`                   | Hide the browser instead of running headed (default during `cypress run`)                                                                                                                     |
```

Comment:
just a placeholder comment to follow up on hiding the reporter

Reply:
OK I confirmed with @jennifer-shehane that we are hiding the reporter. It also sounds like there are going to be new CLI options, `--runner-ui` and -`-no-runner-ui`. We might want to reference that above when we say we removed it and add those options here.